pkg/apis/admissionregistration/types.go

```diff
@@ -165,6 +165,32 @@ type ExternalAdmissionHook struct {
 	// allowed values are Ignore or Fail. Defaults to Ignore.
 	// +optional
 	FailurePolicy *FailurePolicyType
+
+	// Selects namespaces that are subjected to this webhook.
```

Comment:
We should probably reword this. It might be better to be slightly repetitive with the description so it is clear that it applies to both selectors. Something like "Selects whether to run the webhook on an object based on whether the namespace for that object is/is not labeled with the associated key and the values equals any of the provided values. To run the webhook on any object whose namespace was not associated with 'runlevel' of '0' or '1'; you would set the selector as follows."

"If instead you wanted to only run the the webhook on any objects whose namespace was associated with the 'environment' of 'prod' or 'staging'; you would set the selector as follows."

Reply:
>  whether the namespace for that object is/is not labeled with the associated key and the values equals any of the provided values.

This is not accurate, because selector also supports operations like "if key exists/not exists".

I'll just say "whether the namespace for that object matches the selector".

Also, is it intentional to use the past tense?

Updated the comments in types.go, PTAL. Thank you.